src/App.tsx

```diff
@@ -1,5 +1,14 @@
 import React from 'react';
-import './App.css';
+import classNames from 'classnames';
+
+import './App.scss';
+
+import Button from 'react-bootstrap/Button';
+import ToggleButtonGroup from 'react-bootstrap/ToggleButtonGroup';
+import ToggleButton from 'react-bootstrap/ToggleButton';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+import Accordion from 'react-bootstrap/Accordion';
```

Comment:
можно ли импортировать это одним импортом?

Reply:
Yes, but bootstrap recommends to import by element not whole lib

Reply:
"You should import individual components like: react-bootstrap/Button rather than the entire library. Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client."

Reply:
import { Button, ToggleButtonGroup, ToggleButton, OverlayTrigger, Tooltip, Accordion } from 'react-bootstrap';

May be like that?